app/code/Magento/Newsletter/view/frontend/templates/subscribe.phtml

```diff
@@ -8,7 +8,7 @@
 
 ?>
 <div class="block newsletter">
-    <div class="title"><strong>Newsletter</strong></div>
+    <div class="title"><strong><?php /* @escapeNotVerified */ echo __('Newsletter') ?></strong></div>
```

Comment:
I reached out to the core team regarding using `@escapeNotVerified` in backports because it wasn't defined how to handle this.

They asked us to not use `@escapeNotVerified` for new code, even in `2.1-develop` abd when other strings in the same template use the same technique because we know by now that this potentially unsecure.

If you adjust your PR to escape the string (like in [2.2-develop](https://github.com/slackerzz/magento2/blob/2.2-develop/app/code/Magento/Newsletter/view/frontend/templates/subscribe.phtml#L13)) though, I can approve your PR!